smart-campus-service-impl/src/main/java/hu/unideb/smartcampus/service/ejabberd/SharedRosterServiceImpl.java

```diff
@@ -127,8 +124,7 @@ public void deleteUserFromGroup(String user, String group) {
         .host(this.getXmppDomainPropertyValue())
         .group(group)
         .build();
-    return this.clientResponseProvider.sendPostRequest(SHARED_ROSTER_GROUP_INFO_COMMAND, informationRequest)
-        .readEntity(MAP_GENERIC_TYPE);
+    return (Map<String, String>) this.clientResponseProvider.sendPostRequest(SHARED_ROSTER_GROUP_INFO_COMMAND, informationRequest).getEntity();
```

Comment:
Test thrown exception for readEntity, "Exception in thread "main" java.lang.IllegalStateException: Method not supported on an outbound message. "

I found this kind of workaround for this problem so I had to replace it with getEntity.